### ISSUE TITLE
Add auto clearing of all light blocks in a world

### ISSUE THREAD

**Issue by Clyde6790pGIT** [Tschipcraft/dynamiclights#48]
### Is your feature request related to a problem? Please describe.

Yes, sometimes the datapack leaves unused light blocks everywhere when the server stops.

### Describe the feature you'd like

Add an option to automatically clear all light blocks in a world and let the user set the interval

### Describe alternatives you've considered

I've tried using fawe with the /replacenear command, but sometimes it doesn't clear everything.

**Comment by Tschipcraft** [comment]
Are you still experiencing the issue you mentioned in #25 even though you have configured EntityTrackerFixer not to clear the markers, or is this a general request to make it easier to clean up the light blocks if something like this happens again?

**Comment by Clyde6790pGIT** [comment]
> Are you still experiencing the issue you mentioned in #25 even though you have configured EntityTrackerFixer not to clear the markers, or is this a general request to make it easier to clean up the light blocks if something like this happens again?

Yes, there isn't that much unused light blocks anymore ever since I cleaned them up but occasionally when the server crashes or a player just leaves the light they emit does not disappear, and just becomes an unused block that isn't connected to anything, and a cleanup system included would be useful.

**Comment by Tschipcraft** [comment]
Thanks for letting me know. An in game cleanup system isn't really feasible to implement since it would have to scan all chunks for leftover light blocks, but leaving behind light blocks without markers really shouldn't be happening, so I'll have to experiment around a bit to find the underlying issue for this. (Could very well be paper again, since they implemented an async save system where blocks and entities are disconnected a while ago if I'm not mistaken)

**Comment by Clyde6790pGIT** [comment]
Hi @Tschipcraft , is this issue fixed? I cleared up the light blocks after 2 months of not doing so and more than 13k were found and automatically cleaned up, but it cant get all of them.

**Comment by Tschipcraft** [comment]
No, I haven't been able to reproduce this problem (that's why I added the "help wanted" label).
Maybe it's an incompatibility with some other plugin instead? Could you send me a list of which additional plugins you are using (ideally including any configuration changes to them as well)? Maybe I can find the cause then.

**Comment by Clyde6790pGIT** [comment]
@Tschipcraft , I tried running it on a paper server with no plugins or datapacks other than this one, it still happens, but it works fine on vanilla Minecraft, perhaps this might be with paper.

**Comment by Tschipcraft** [comment]
Interesting, I couldn't reproduce it on a plain paper server. Did you change some configuration values from the defaults? Another possibility could be that I didn't monitor long enough?
